app/helpers/riiif_helper.rb

```diff
@@ -38,7 +38,7 @@ def riiif_image_srcset_pixel_density(riiif_file_id, base_width, format: 'jpg', q
   #
   # if use_image_server is false, size_key is ignored and no srcsets are generated,
   # we just use the stock hydra-derivative created image labelled 'jpeg'
-  def member_image_tag(member, size_key: nil, lazy: false, use_image_server: CHF::Env.lookup(:use_image_server_on_show_page))
+  def member_image_tag(parent_id, member, size_key: nil, lazy: false, use_image_server: CHF::Env.lookup(:use_image_server_on_show_page))
     base_width = size_key == :large ? 525 : 208
```

Comment:
gah, I guess we have to pass parent_id in too now for the analytics. :(  Should we make them all keyword args to avoid confusion as to which is which?